src/murmur/Meta.cpp

```diff
@@ -50,7 +50,7 @@ MetaParams::MetaParams() {
 	qsPassword                 = QString();
 	usPort                     = DEFAULT_MUMBLE_PORT;
 	iTimeout                   = 30;
-	iMaxBandwidth              = 72000;
+	iMaxBandwidth              = 558000; // 558000 = 510000 (Opus) + 9600 (position) + 38400 (TCP overhead)
```

Comment:
```suggestion
	// This value represents the maximum possible bandwidth requirement using 10ms (restricted
	// by the maximum Opus supports)
	// 558000 = 510000 (Opus) + 9600 (position) + 38400 (TCP overhead)
	iMaxBandwidth              = 558000; 
```
Is that explanation even correct? :eyes: 
I think it is important to state something like that as well though :point_up:

Reply:
Added an explanation and ran clang-format.